scripts/release-notes/gen_ec_release_notes.py

```diff
@@ -265,7 +306,7 @@ def find_new_releases(url_base, release_type):
         # Skip very old RCs, indicated by the first 2 parts of the
         # version string major.minor.
         version_prefix = '.'.join(version.split('.')[:2])
-        if version_prefix in OLD_VERSIONS:
+        if version_prefix not in versions_to_scan:
```

Comment:
`OLD_VERSIONS` looks unused after this change

Reply:
Good catch, removed